smoothie/build.gradle

```diff
@@ -1,42 +1,51 @@
 buildscript {
   repositories {
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
+    jcenter()
+    google()
   }
   dependencies {
-    classpath "gradle.plugin.org.inferred:gradle-processors:1.2.1"
+    classpath ("com.android.tools.build:gradle:3.0.0-beta2")
   }
+
 }
 
-apply plugin: 'java'
-apply plugin: 'org.inferred.processors'
+apply plugin: 'com.android.library'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+dependencies {
+  api project(':toothpick')
+  api deps.inject
```

Comment:
smoothie is an optional module including only bindings (and some providers used by them).

i think these 2 dependencies should be compileOnly too as `inject` and `project` should be already there